ui/src/environments/environment.perf.ts

```diff
@@ -13,6 +13,7 @@ export const environment: Environment = {
   gaId: 'UA-112406425-2',
   gaUserAgentDimension: 'dimension1',
   gaLoggedInDimension: 'dimension2',
+  gaUserInstitutionCategoryDimension: 'dimension3',   // TODO create
```

Comment:
I haven't created these yet in GA.  Reminder: I need to do that and update this text